pipeline.py

```diff
@@ -112,32 +91,32 @@ def pipeline(
-        sdg_pipeline: str = SDG_PIPELINE,
-        sdg_max_batch_len: int = MAX_BATCH_LEN,
-        sdg_sample_size: float = 1.0,
+        sdg_scale_factor: int = 30,  # https://github.com/instructlab/instructlab/blob/d68c5a12026e08092bd4171a77667b09f7c36f72/tests/testdata/default_config.yaml#L125
+        sdg_pipeline: str = "full",  # https://github.com/instructlab/instructlab/blob/d68c5a12026e08092bd4171a77667b09f7c36f72/tests/testdata/default_config.yaml#L122
+        sdg_max_batch_len: int = 5000,  # https://github.com/instructlab/instructlab/blob/d68c5a12026e08092bd4171a77667b09f7c36f72/tests/testdata/default_config.yaml#L334
+        sdg_sample_size: float = 1.0,  # FIXME: Not present in default config. Need justification?
         # Training phase
-        train_nproc_per_node: int = 3,
-        train_nnodes: int = 2,
-        train_num_epochs_phase_1: int = NUM_EPOCHS_PHASE_1,
-        train_num_epochs_phase_2: int = NUM_EPOCHS_PHASE_2,
-        train_effective_batch_size_phase_1: int = EFFECTIVE_BATCH_SIZE_PHASE_1,
-        train_effective_batch_size_phase_2: int = EFFECTIVE_BATCH_SIZE_PHASE_2,
-        train_learning_rate_phase_1: float = LEARNING_RATE_PHASE_1,
-        train_learning_rate_phase_2: float = LEARNING_RATE_PHASE_2,
-        train_num_warmup_steps_phase_1: int = NUM_WARMUP_STEPS_PHASE_1,
-        train_num_warmup_steps_phase_2: int = NUM_WARMUP_STEPS_PHASE_2,
-        train_save_samples: int = SAVE_SAMPLES,
-        train_max_batch_len: int = MAX_BATCH_LEN,
-        train_seed: int = SEED,
+        train_nproc_per_node: int = 3,  # FIXME: Not present in default config. Need justification?
+        train_nnodes: int = 2,  # FIXME: Not present in default config. Need justification?
+        train_num_epochs_phase_1: int = 7,  # https://github.com/instructlab/instructlab/blob/d68c5a12026e08092bd4171a77667b09f7c36f72/tests/testdata/default_config.yaml#L364
+        train_num_epochs_phase_2: int = 10,  # https://github.com/instructlab/instructlab/blob/d68c5a12026e08092bd4171a77667b09f7c36f72/tests/testdata/default_config.yaml#L377
+        train_effective_batch_size_phase_1: int = 128,  # https://github.com/instructlab/instructlab/blob/d68c5a12026e08092bd4171a77667b09f7c36f72/tests/testdata/default_config.yaml#L357
+        train_effective_batch_size_phase_2: int = 3840,  # https://github.com/instructlab/instructlab/blob/d68c5a12026e08092bd4171a77667b09f7c36f72/tests/testdata/default_config.yaml#L371
+        train_learning_rate_phase_1: float = 2e-05,  # https://github.com/instructlab/instructlab/blob/d68c5a12026e08092bd4171a77667b09f7c36f72/tests/testdata/default_config.yaml#L360
+        train_learning_rate_phase_2: float = 6e-06,  # https://github.com/instructlab/instructlab/blob/d68c5a12026e08092bd4171a77667b09f7c36f72/tests/testdata/default_config.yaml#L374
+        train_num_warmup_steps_phase_1: int = 100,  # FIXME: Not present in default config. Need justification?
```

Comment:
Not exposed by `ilab` CLI but present in the training lib https://github.com/instructlab/training/blob/main/src/instructlab/training/main_ds.py#L874